src/compose.js

```diff
@@ -7,12 +7,13 @@ const assign = Object.assign;
 
 /**
  * Creates new factory instance.
- * @param {object} descriptor The information about the object the factory will be creating.
+ * @param {Descriptor} descriptor The information about the object the factory will be creating.
  * @returns {Function} The new factory function.
  */
 function createFactory(descriptor) {
   return function Stamp(options, ...args) {
-    const obj = Object.create(descriptor.methods || {});
+    // Next line was optimized for most JS VMs. Please, be careful here!
+    const obj = Object.create(descriptor.methods || null);
```

Comment:
@FredyC - Because of V8's hidden classes optimization, using new can be up to **10 times faster** for property access (not instantiation) than `Object.create()`.

Of course we should profile both instantiation and property access. We also need to run the check and loop about a million times to see all the optimizations kick in -- on modern CPUs, that shouldn't take terribly long.